andela/travel_tool_front
# #160720181 Fix edit comment button behaviour

#### What does this PR do?
This PR fixes on hover, Save button color and the edited label bug issues

#### Description of Task to be completed?
- Fix on hover to change the color of edit button to blue
- Fix the color of the Save button so that it is blue when the edit button is clicked and the comment box opens
- Fix the edited label so that it is only visible if a comment was actually updated/edited

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the request Id from the list of requests displayed on the request table, you should be able to see the Request details Modal displayed.
- Add a new comment using the comment box.
- Hover over the edit button, it should turn blue and underlined
- On clicking the edit button, the comment box opens with the save button having color blue
- Go ahead and click on the save button without editing the comment, the comment box closes and the comment displayed should not have edited label attached to it.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160720181](https://www.pivotaltracker.com/story/show/160720181)

#### Screenshots (if appropriate)
Screenshot showing the save button color blue when the edit button is clicked.
<img width="1276" alt="screen shot 2018-09-26 at 10 38 42" src="https://user-images.githubusercontent.com/12033494/46064898-5705c680-c179-11e8-8dfd-84dbb3629547.png">

Screenshot showing the save button disabled when there is no text in the commentbox
<img width="1280" alt="screen shot 2018-09-26 at 10 38 22" src="https://user-images.githubusercontent.com/12033494/46064920-638a1f00-c179-11e8-997a-9268c6e63d64.png">




#### Questions:
None
